README.md

```diff
@@ -14,7 +13,13 @@ See the plugin documentation for theme installation guidelines.
 
 ## Screenshots
 
-Coming soon!
+Main screen:
+
+![Main screen](./docs/images/screenshot_main.png)
+
+Build console:
+
+![Build console log](./docs/images/screenshot_build_console.png)
```

Comment:
this doesn't render, it's saved as a .PNG (capitals)

either:
```suggestion
![Build console log](./docs/images/screenshot_build_console.PNG)
```

or rename the file

Reply:
Thanks Windows! Let's keep it as is, the screenshots are temporary anyway